sympy/utilities/tests/test_lambdify.py

```diff
@@ -362,10 +362,17 @@ def test_numpy_old_matrix():
     assert isinstance(f(1, 2, 3), numpy.matrix)
 
 def test_python_div_zero_issue_11306():
+    # The Try:Except condition is put to check whether zero division returns answer or not.
     if not numpy:
         skip("numpy not installed.")
-    p = Piecewise((1 / x, y < -1), (x, y <= 1), (1 / x, True))
-    lambdify([x, y], p, modules='numpy')(0, 1)
+    p = Piecewise((1 / x, y < -1), (x, y < 1), (1 / x, True))
+    f = lambdify([x, y], p, modules='numpy')
+    numpy.seterr(all='raise')
```

Comment:
I guess `numpy.seterr(all='ignore')` will not check whether the input produces an output or not.
I set it to raise so that I can catch the raised warning and check the output which ensures that the wrapper converts properly and produces `'inf'` as output.